packages/gatsby-plugin-cxs/.babelrc

```
@@ -0,0 +1,7 @@
+{
+  "presets": [
+    ["../../.babel-preset.js", {
```

```suggestion
    ["babel-preset-gatsby-package", {
```

we recently added this, so sorry for the churn here!

---

No worry 😄